test/pytest/test_linters.py

```diff
@@ -0,0 +1,15 @@
+import glob
+import os
+import subprocess
+import sys
+
+import pytest
+
+SRCDIR = os.path.realpath(f'{__file__}/../../..')
+
+
+@pytest.mark.parametrize('linter', ['flake8', 'pycodestyle', 'pylint', 'vulture'])
+@pytest.mark.parametrize('filename', glob.glob('src/cockpit/**/*.py', root_dir=SRCDIR, recursive=True))
```

Comment:
In other words, this glob shouldn't be a test parameter, but an argument to calling the linter -- or even better, they normally should detect the modules by themselves.

You explicitly don't want to run them on executables here?